### DISCUSSION TITLE
Installing firebase creates @firebase and firebase node_modules

### DISCUSSION THREAD

**Discussion by MauriceArikoglu** [firebase/firebase-js-sdk#6342] [General]
Why does npm i firebase cause `@firebase` and `firebase` to be created in node_modules?

**Comment by akauppi** [comment]
This is normal. `firebase` is the umbrella package used for versioning the releases. Its dependencies are the `@firebase` packages.

You can find this information by looking at `node_modules/firebase/package.json`.
